src/js/build.js

```diff
@@ -2,12 +2,31 @@ import axios from 'axios'
 import JSZip from 'jszip'
 import Sass from 'sass.js/dist/sass'
 import CleanCSS from './lib/clean-css'
+import autoprefixer from './lib/autoprefixer'
 
 import { jsPlugins, scssPlugins } from './plugins'
 import { createJsFileContent, generateLink } from './file-util'
 import { formatScssList, uniqArray } from './util'
 
 const popperCDN = 'https://unpkg.com/popper.js/dist/umd/popper.js'
+const autoPrefixerConfig = {
+  cascade: false,
+  from: undefined,
```

Comment:
Is `from` really needed?

Reply:
yep without that we have a warning in the `console`

Reply:
Can it be just an empty string?

Reply:
the warning said, to avoid that we should use `undefined`

Reply:
OK, I guess, I just don't get what is throwing the warning :P

Reply:
![image](https://user-images.githubusercontent.com/1689750/46600089-6249e200-cae9-11e8-8da5-5f3d8946bb05.png)

Reply:
Ah, OK, makes sense now. All good then.